docs/src/model_creation/reactionsystem_content_accessing.md

```diff
@@ -0,0 +1,249 @@
+
+Let us consider the following [two-state model](@ref basic_CRN_library_two_states)
+```@example model_accessing_symbolic_variables
+using Catalyst
+rs = @reaction_network begin
+    (k1,k2), X1 <--> X2
+end
+```
+If we wish to access one of the symbolic variables stored in it (here `X1`, `X2`, `k1`, and `k2`), we simply write 
+```@example model_accessing_symbolic_variables
+rs.X1
+```
+to access e.g. `X1`. This symbolic variable can be used just like those [declared using `@parameters` and `@species`](@ref programmatic_CRN_construction):
+```@example model_accessing_symbolic_variables
+using OrdinaryDiffEq
+u0 = [rs.X1 => 1.0, rs.X2 => 2.0]
+ps = [rs.k1 => 2.0, rs.k2 => 4.0]
+oprob = ODEProblem(rs, u0, (0.0, 10.0), ps)
+sol = solve(oprob)
+nothing # hide
+```
+We can also use them to form symbolic expressions:
+```@example model_accessing_symbolic_variables
+Xtot = rs.X1 + rs.X2
+```
+which can be used when we e.g. [plot our simulation](@ref simulation_plotting_options):
+```@example model_accessing_symbolic_variables
+using Plots
+plot(sol; idxs = [rs.X1, rs.X2, Xtot])
+```
+
+Next we create our two-state model [programmatically](@ref programmatic_CRN_construction):
+```@example model_accessing_symbolic_variables
+t = default_t()
+@species X1(t) X2(t)
+@parameters k1 k2
+rxs = [
+    Reaction(k1, [X1], [X2]),
+    Reaction(k2, [X2], [X1])
+]
+@named rs_prog = ReactionSystem(rxs, t)
+rs_prog = complete(rs_prog)
+nothing # hide
+```
+Here, we can confirm that the symbolic variables we access through the model are identical to those we used to create it:
+```@example model_accessing_symbolic_variables
+isequal(rs.k1, k1)
+```
+
+!!! warn
+    When accessing model symbolic variables through the model (using e.g. `rs.X1`), it is important to first ensure that the [*model is complete*](@ref programmatic_CRN_construction).
+
+## [Accessing basic model properties](@id model_accessing_basics)
+
+### [Accessing model parameter and species](@id model_accessing_basics_parameters_n_species)
+Previously we showed how to access individual parameters or species of a `ReactionSystem` model. Next, the `parameters` and `species` functions allow us to retrieve *all* parameters and species as vectors:
```

Comment:
```suggestion
Previously we showed how to access individual parameters or species of a `ReactionSystem` model. Next, the [`parameters`](@ref) and [`species`](@ref) functions allow us to retrieve *all* parameters and species as vectors:
```
All these API functions you are introducing should be links to their documentation. (I haven't made comments beyond this point, but it would be good to update them below too.)

Reply:
will do